samza-core/src/main/java/org/apache/samza/config/TaskConfigJava.java

```diff
@@ -37,7 +37,7 @@
 public class TaskConfigJava extends MapConfig {
   // Task Configs
   private static final String TASK_SHUTDOWN_MS = "task.shutdown.ms";
-  public static final long DEFAULT_TASK_SHUTDOWN_MS = 5000L;
+  public static final long DEFAULT_TASK_SHUTDOWN_MS = 30000L;
```

Comment:
@sborya I think we missed to discuss about this comment yesterday. Another thing I noticed is that `TaskConfigJava` and `TaskConfig` have different defaults for `task.shutdown.ms`. Can you please fix this and also, comment on what you think about the impact on existing jobs in yarn that may rely on this default ? 
If we are changing this, we should change it everywhere in the code, update the config documentation and also, enlist it in the release notes.